content/v2.0/application/settings.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Settings
+order: 10
+---
+
+Hanami supports application settings through a `Settings` class, defined in `config/settings.rb`.
```

Comment:
I have several thoughts about this very first line 😆 

First, I think it's good to establish a shorthand term for this that we can use throughout all our guides. You're close here with "application settings," but since we call our app an `Hanami::App` and it's defined in app.rb, perhaps "app settings" is better.

We'll also want to make sure we're clear in our language so this is not confused with framework configuration (i.e. `Hanami.app.config` or `SomeSlice::Slice.config`). I think calling these "app config" or "slice config" (i.e. config as distinct from _settings_) would be the way to go.

None of this is immediately applicable to this first line, apart perhaps considering "app" instead of "application" for our vernacular — I'm keen for your thoughts on this, however.

The actual thing I think we can change about this firstly line is to more strongly emphasise that these are _user defined_ settings, so perhaps something like this:

> You can define your own settings for your app through a `Settings` class, defined in `config/settings.rb`.

wdyt?

Reply:
Yep, excellent distinction to make clear. I've gone with your opening line, followed by:

```
Using this class, you can specify what settings exist within your application, what types and defaults they have, and whether or not they are required for your application to boot.

These "app settings" are unrelated to ["app config"](/v2.0/application/config/), which configure framework behaviours. App settings are your own to define and use.
```